src/Microsoft.AspNetCore.AngularServices/Properties/AssemblyInfo.cs

```diff
@@ -1,12 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Reflection;
-using System.Resources;
-using System.Runtime.CompilerServices;
-
-[assembly: AssemblyMetadata("Serviceable", "True")]
```

Comment:
Unless you add Internal.AspNetCore.Sdk, you'll need to keep this one. It's not generated by Microsoft.NET.Sdk.

cref https://github.com/aspnet/BuildTools/blob/e4be7e105de6c3bf2e882bb76f3565305ca5a46d/src/Internal.AspNetCore.Sdk/build/GenerateAssemblyInfo.targets